nautobot/core/events/__init__.py

```diff
@@ -12,6 +16,32 @@
 logger = logging.getLogger(__name__)
 
 
+def load_event_brokers(settings):
+    """Process plugins and log errors if they can't be loaded."""
+    if not getattr(settings, "NAUTOBOT_EVENT_BROKERS", None):
```

Comment:
We should define an appropriate default (empty?) value in `settings.py` and assert that it's always present.